### PR TITLE
added prevalence period to symptom. this replaces onset and abatement.

### PR DESCRIPTION
**Pull request by edeyoung**
There was a discrepancy between the model-info file and the model. This change addresses this.

Pull requests into CQL QDM Patient API require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] JIRA ticket for this PR: https://jira.mitre.org/browse/BONNIE-1163
- [x] JIRA ticket links to this PR
- [x] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [x] If UI changes have been made, google WAVE plug-in has been executed to ensure no 508 issues were introduced.  N/A
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass (remember to update Gemfile when applicable)
- [x] Code coverage has not gone down and all code touched or added is covered.
     * In rare situations, this may not be possible or applicable to a PR. In those situations:
         1. Note why this could not be done or is not applicable here:
         2. Add TODOs in the code noting that it requires a test
         3. Add a JIRA task to add the test and link it here:
- [x] Automated regression test(s) pass

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA test links: N/A
- [x] Justification for using JIRA tests: N/A
- [x] JIRA tests have been added to sprint N/A

**Reviewer 1:**

Name: @mayerm94
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree

**Reviewer 2:**

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree

======

Code coverage results:

cql4bonnie: 

```
=============================== Coverage summary ===============================
Statements   : 57.05% ( 1153/2021 )
Branches     : 26.29% ( 173/658 )
Functions    : 41.89% ( 191/456 )
Lines        : 53.06% ( 953/1796 )
================================================================================
```

this branch

```
=============================== Coverage summary ===============================
Statements   : 57.5% ( 1165/2026 )
Branches     : 26.74% ( 177/662 )
Functions    : 42.42% ( 193/455 )
Lines        : 53.58% ( 965/1801 )
================================================================================
```

### CODE REVIEW

File: app/assets/javascripts/datatypes/symptom.js.coffee [hunk 1]
```diff
@@ -20,21 +20,24 @@ class CQL_QDM.Symptom extends CQL_QDM.QDMDatatype
...
   constructor: (@entry) ->
     super @entry
-    @_abatementDatetime = CQL_QDM.Helpers.convertDateTime(@entry.abatementDatetime)
-    @_onsetDatetime = CQL_QDM.Helpers.convertDateTime(@entry.start_time)
+    @_prevalencePeriodLow = CQL_QDM.Helpers.convertDateTime(@entry.start_time)
+    if @entry.end_time
+      @_prevalencePeriodHigh = CQL_QDM.Helpers.convertDateTime(@entry.end_time)
+    else
+      # No end time; high is set to infinity
+      @_prevalencePeriodHigh = CQL_QDM.Helpers.infinityDateTime()
```
Comment (mayerm94) [inline app/assets/javascripts/datatypes/symptom.js.coffee]: This line is hit in the test 'should show null prevalence period', it might be worth adding a test that has a null end_time but a value for start_time so that we can assert that the infinitydateTime is used. This would mean there is a comprehensive test for prevalence period that could be copy/pasted into the tests for all datatypes that we talked about doing to ensure compliance with the model_info file
